omod/src/main/java/org/openmrs/module/aijar/fragment/controller/PatientStabilityFragmentController.java

```diff
@@ -167,10 +170,10 @@ public void controller(FragmentModel model, @RequestParam(value = "patientId", r
      * @param noOfMoths
      * @return
      */
-    public String getDateBefore(Date referenceDate, int noOfMoths,int noOfYears) {
+    public String getDateBefore(Date referenceDate, int noOfMoths, int noOfYears) {
```

Comment:
@slubwama Do add comments on each of the methods in this class to describe what each one does